lib/modules/manager/index.ts

```diff
@@ -60,7 +61,7 @@ export function extractPackageFile(
   manager: string,
   content: string,
   fileName: string,
-  config: ExtractConfig
+  config: Partial<WorkerExtractConfig>
```

Comment:
Updated the type of config in `extractPackageFile` cause it handles config of both the types `ExtractConfig` and `CustomExtractConfig` while `extractAllPackageFiles` only handles `ExtractConfig`. TS requires this aftre we have narrowed down the config.